data/2023/2023-12-26/readme.md

```diff
@@ -0,0 +1,233 @@
+# R Package Structure
+
+Happy Boxing Day!
+While you're dealing with your physical packages, we're looking into R packages!
+
+The dataset this week comes from ["Historical Trends in R Package Structure and Interdependency on CRAN"](https://mpadge.github.io/pkgstats-analyses/articles/pkgstats.html) by Mark Padgham and Noam Ross.
+In that paper, they use the [{pkgstats}](https://cran.r-project.org/package=pkgstats) R package to analyze the structure of R packages over time, using an archive of all packages on [CRAN](https://cran.r-project.org/) as of 2022-11-22.
+We've provided csv versions of two of [the datasets from that paper](https://zenodo.org/records/7414296).
+
+The paper focuses on package characteristics over time.
+It might be interesting to look at the distribution of similar features (such as lines of code) across packages.
+
+If you're unfamiliar with some of the terminology in this dataset, you might find the [R Packages](https://r-pkgs.org/) book by Hadley Wickham and Jennifer Bryan helpful.
+
+## The Data
+
+
+```{r}
+# Option 1: tidytuesdayR package 
+## install.packages("tidytuesdayR")
+
+tuesdata <- tidytuesdayR::tt_load('2023-12-26')
+## OR
+tuesdata <- tidytuesdayR::tt_load(2023, week = 52)
+
+cran_20221122 <- tuesdata$cran_20221122
+external_calls_20221122 <- tuesdata$external_calls_20221122
+internal_calls_20221122 <- tuesdata$internal_calls_20221122
```

Comment:
I think this should be 'external_calls' and 'internal_calls', rather than e.g. 'external_calls_20221122'?